types/trigger.d.ts

```diff
@@ -55,3 +55,16 @@ export type ResponseOutput<Matches> = {
 // The type of a response trigger field.
 export type ResponseFunc<Matches> =
     (data: Data, matches: Matches, output: Output) => ResponseOutput<Matches>;
+
+export type Trigger = {
+  id: string;
+  regex: RegExp;
+  beforeSeconds: number;
+}
```

Comment:
Can you leave a TODO here so it's more obvious to other people that this isn't used?